pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -475,8 +577,16 @@ def do_new_test(self,
                 os.remove(self.home_path('network_init.json'))
             except FileNotFoundError:
                 pass
+            try:
+                self.neard_init(rpc_port, protocol_port, validator_id,
+                                genesis_protocol_version,
+                                epoch_config_overrides)
+            except Exception as e:
```

Comment:
Please try to catch a more specific set of exceptions, or to express the logic here in some other way if you want. Catching all exceptions is not great because some of them should really just cause us to exit if they indicate a bug in the program rather than some expected thing

see "Never use catch-all except: statements" [here](https://google.github.io/styleguide/pyguide.html#24-exceptions)